apps/connect/source/ftrack_connect/plugin_manager/plugin_list.py

```diff
@@ -242,33 +241,18 @@ def plugin_is_available(self, plugin_data):
                 return item
         return None
 
-    def _is_plugin_valid(self, plugin_path):
-        '''Return whether the provided *plugin_path* is a valid plugin.'''
-        plugin_name = os.path.basename(plugin_path)
-        match = self.plugin_re.match(plugin_name)
-        if match:
-            data = match.groupdict()
-        else:
-            return False
-
-        data['platform'] = 'noarch'
-        if data['version'].lower().endswith('.zip'):
-            # pop zip extension from the version.
-            # TODO: refine regex to catch extension
-            data['version'] = data['version'][:-4]
-            parts = data['version'].split('-')
-            if len(parts) > 1:
-                data['version'] = parts[0]
-                data['platform'] = parts[-1]
-
-        return data
-
     def populate_installed_plugins(self):
         '''Populate model with installed plugins.'''
         self._installed_plugin_count = 0
         self._plugin_model.clear()
 
-        plugins = os.listdir(self.default_plugin_directory)
+        # Filter out files and hidden items.
+        plugins = [
```

Comment:
Check if hook folder exists as well? rules out random folders.

Reply:
I think that should be checked on the incompatible / deprecated/etc plugins. We can have a task to improve that part also and update it as I did in the incompatible ones.